From 1bd05f78732e29d90cb9ad89412fae45fdbaea7b Mon Sep 17 00:00:00 2001
From: cmitu <31816814+cmitu@users.noreply.github.com>
Date: Mon, 20 Jan 2025 17:21:22 +0000
Subject: [PATCH] libretro: fix build breakage

The previous commit on sdl/scrnmgr.c broke the _libretro_ build, add an additional '#ifdef' for a codepath that's not supposed to be used by the libretro core.
---
 sdl/scrnmng.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/sdl/scrnmng.c b/sdl/scrnmng.c
index 1d59b5bd..e138baaa 100755
--- a/sdl/scrnmng.c
+++ b/sdl/scrnmng.c
@@ -156,6 +156,7 @@ BRESULT scrnmng_create(UINT8 mode) {
 	scrnsurf.extend = 0;																// ?
 
 	scrnmng.enable = TRUE;
+#if !defined(__LIBRETRO__)
 	#if SDL_MAJOR_VERSION != 1
 		if((mode & SCRNMODE_FULLSCREEN) && !scrnmng_isfullscreen()) {
 			scrnmng.flag |= SCRNFLAG_FULLSCREEN;
@@ -166,6 +167,7 @@ BRESULT scrnmng_create(UINT8 mode) {
 			}
 		}
 	#endif
+#endif    /* __LIBRETRO__ */
 	return(SUCCESS);
 }
 
